sdk/storage/storage-blob/src/BlockBlobClient.ts

```diff
@@ -207,6 +259,15 @@ export interface BlockBlobCommitBlockListOptions {
    * @memberof BlockBlobCommitBlockListOptions
    */
   metadata?: Metadata;
+  /**
+   * Customer Provided Key Info.
+   *
+   * @type {Models.CpkInfo}
+   * @memberof BlockBlobCommitBlockListOptions
+   */
+  customerProvidedKey?: Models.CpkInfo;
+  accessTier?: Models.AccessTier;
+  tier?: BlockBlobTier | string;
```

Comment:
This came in from the master branch.
https://github.com/Azure/azure-sdk-for-js/blob/c2b39e494c36d8b8f7d6de91c87b04937b283911/sdk/storage/storage-blob/src/BlockBlobURL.ts#L69

**accessTier vs tier**
```
export type AccessTier = 'P4' | 'P6' | 'P10' | 'P15' | 'P20' | 'P30' | 'P40' | 'P50' | 'P60' | 'P70' | 'P80' | 'Hot' | 'Cool' | 'Archive';
```
```
export enum BlockBlobTier {
  Hot = "Hot",
  Cool = "Cool",
  Archive = "Archive"
}

export enum PremiumPageBlobTier {
  P4 = "P4",
  P6 = "P6",
  P10 = "P10",
  P15 = "P15",
  P20 = "P20",
  P30 = "P30",
  P40 = "P40",
  P50 = "P50",
  P60 = "P60",
  P70 = "P70",
  P80 = "P80"
}
```

We probably don't need both `accessTier` and `tier` as arguments. 
Also, there is no test that uses either of these options.

/cc - @jeremymeng @jiacfan

Reply:
@jiacfan Please help confirm

Reply:
Great catch, only `tier` is useful, please remove `accessTier`. There is test case to cover it, please check [here](https://github.com/Azure/azure-sdk-for-js/blob/c2b39e494c36d8b8f7d6de91c87b04937b283911/sdk/storage/storage-blob/test/blockbloburl.spec.ts#L200). At same time will create a issue in master branch to cover it, and fix it in next release.

Reply:
d34c0fa